rest/service/src/main.rs

```diff
@@ -9,12 +10,19 @@ use rustls::{
 use std::io::BufReader;
 use structopt::StructOpt;
 
+/// uri where the openapi spec is exposed
+pub(crate) const SPEC_URI: &str = "/v0/api/spec";
+
 #[derive(Debug, StructOpt)]
-struct CliArgs {
-    /// The Rest Server address to bind to
+pub(crate) struct CliArgs {
+    /// The Rest https Server address to bind to
```

Comment:
```suggestion
    /// The bind address for the REST interface (with HTTPS)
```

[Wikipedia](https://en.wikipedia.org/wiki/Representational_state_transfer) suggests 'Rest' (which I presume refers to Representational state transfer) be all caps: REST.